conda-recipe/meta.yaml

```diff
@@ -14,96 +14,105 @@
+# NB: this recipe should be synced between sklearnex repo and feedstocks
+
+{% set name = "scikit-learn-intelex" %}
+{% set version = "2025.1" %}
 {% set buildnumber = 0 %}
 
 package:
-    name: daal4py
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
+# Reminder: source should point to path for repo or github archive for feedstock
+# source:
+#   url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
+#   sha256: ...
 source:
-    path: ..
+  path: ..
 
 build:
-    number: {{ buildnumber }}
-    include_recipe: False
-    script_env:
-    - DPCPPROOT
-    - DALROOT
-    ignore_run_exports:
-       - python
-       - mpich
+  skip: true  # [not (linux64 or win)]
+  number: {{ buildnumber }}
+  include_recipe: False
+  script_env:
+  - DPCPPROOT
+  - DALROOT
+  - NO_DIST=1  # [win]
 
 requirements:
-    build:
-        - {{ compiler('cxx') }}  # [not osx]
-        - {{ compiler('c') }}  # [not osx]
-    host:
-        - python
-        - setuptools
-        - numpy {{ numpy }}
-        - dal-devel ==2024.1.0
-        - cython
-        - jinja2
-        - mpich  # [osx]
-        - impi-devel  # [not osx]
-        - clang-tools
-        - pybind11
-        - make
-    run:
-        - python
-        - dpcpp_cpp_rt ==2024.1.0  # [linux]
-        - dpcpp_cpp_rt ==2024.1.0  # [win or osx]
-        - dal ==2024.1.0
-    ignore_run_exports:
-        - numpy
+  build:
+    - make  # [linux]
+    - dpcpp_linux-64  # [linux64]
+    # - dpcpp_win-64  # [win]
+    - {{ compiler('cxx') }}  # [linux64 or win]
+    - {{ compiler('c') }}  # [linux64 or win]
+    # conda-forge feedstock specific
+    # - {{ stdlib("c") }}  # [linux64 or win]
+  host:
+    - python
+    - setuptools
+    - cmake
+    - clang-format
+    - cython
+    - jinja2
+    - pybind11
+    - numpy {{ numpy }}
+    - mpich  # [not win]
+    # - impi-devel  # [win]
+    # dal-devel pinning depends on the recipe location (repo or feedstock)
+    - dal-devel
+    # - dal-devel =={{ version }}
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - dpcpp-cpp-rt  # [linux64]
+    # dal pinning depends on the recipe location (repo or feedstock)
+    - dal
+    # - dal =={{ version }}
 
 test:
-    requires:
-        - lightgbm  # [not win]
-        - pandas
-        - scipy
-        - scikit-learn
-        - threadpoolctl
-        - xgboost  # [not win]
-        - catboost
-        - pytest
-        - mpich  # [osx]
-        - impi_rt  # [not osx]
-    source_files:
-        - examples
-        - tests
-        - daal4py/sklearn
-    commands:
-        - python -c "import daal4py"
-        - mpirun -n 4 python -m unittest discover -v -s tests -p test*spmd*.py # [not win]
-        - mpiexec -localonly -n 4 python -m unittest discover -v -s tests -p test*spmd*.py # [win]
-        - python -m unittest discover -v -s tests -p test*.py
-        - pytest --pyargs daal4py/sklearn/
-        - python tests/run_examples.py
+  requires:
```

Comment:
Actually now that I think of it, is ```tests:``` evaluated at all in ci?  I see that the azure pipelines job will build it, but do we try to evaluate the testing side of meta.yml?  If no, then I wouldn't do the full testing suite. To go off my previous review comment, I think model builders would best test the ```requres:``` list.

Reply:
`conda-build` seems to prefer `run_test.bat / run_test.sh` over test commands in recipe, so full testing is included in conda-recipe CI by default.

Reply:
@Alexsandruss thank you for the note, this is what impacts #2107 and #2104 @david-cortes-intel